pkg/reconciler/v1alpha1/eventlistener/eventlistener_test.go

```diff
@@ -1045,6 +1072,24 @@ func TestReconcile(t *testing.T) {
 			ConfigMaps:     []*corev1.ConfigMap{loggingConfigMap},
 		},
 	},
+		{
```

Comment:
This test case verifies the intended behaviour initially stated here:
https://github.com/tektoncd/triggers/issues/318

Reply:
Nice, I think this is also related to #586